### DISCUSSION TITLE
[Suggestion]: Possible to add config for more default objectives (higher than 2)?

### DISCUSSION THREAD

**Discussion by Hidden44** [ejektaflex/Bountiful#269] [Mod Ideas]
Hey, I have bounties with 3 rewards set, but I only see at most 2 objectives. I would like to see an config to set default objective limit to 3 or something.

Yes, I know how to balance rewards/objectives, I just want to see 2-3 objectives very commonly, not 1-2.

**Comment by ejektaflex** [comment]
Hi @Hidden44 ! I'll take this into consideration. Right now, the bounty creation algorithm *must* generate a balanced bounty, and as such I can not set a hard limit on objectives (because if someone created a Decree with only dirt as an objective and only diamonds as a reward and put limits on the number of rewards AND the number of objectives, then suddenly the algorithm would no longer be able to do it's job).

However, there is still a built in 'preference' for 1-2 objectives. I'd be willing to add a config option for the preference range for when it comes to generating objectives, so that you can see 2-3 more often than 1-2. :)

Granted, this will be a *preference*, the algorithm will still add more objectives if it deems it necessary. But, you know, if youre objs/rews are balanced then that shouldn't ever be a problem anyway.

I've added this to my to-do list!

**Reply by Hidden44** [reply]
Cool~ Thank you~

**Comment by ejektaflex** [comment]
Added in 25e14dfc7f7c6e86d9cc4408dbfbc26cad597310 - this was included in the release for 1.20.4. Enjoy!
